README.md

```diff
@@ -2,8 +2,147 @@
 
 [![Build Status](https://travis-ci.org/hmrc/file-transmission.svg)](https://travis-ci.org/hmrc/file-transmission) [ ![Download](https://api.bintray.com/packages/hmrc/releases/file-transmission/images/download.svg) ](https://bintray.com/hmrc/releases/file-transmission/_latestVersion)
 
-This is a placeholder README.md for a new repository
+Microservice that facilitates transmission of files between MDTP and MDG.
+
+Whenever service on MDTP platform wants to send bundle (batch) of files to MDG it will need to use this service. The file-transmission
+service notifies MDG that files prepared by service on MDTP are ready to be processed. Please note that file-transmission service 
+do not transfer file contents. The service just passes data about file location. The file content has to be provided by other service (eg. upscan)
+
+# Typical usage scenario
+
+- Consuming service requests upload of the file from user using `upscan` service
+- `Upscan` notifies consuming service that the file has been uploaded and is ready to be processed - giving consuming 
+service the URL that allows to download the file.
+- Consuming service ensures that all required files have been uploaded by the user
+- Consuming service uses file-transmission service to notify MDG that files are ready to be processes
+- MDG asynchronously uses the file
+
```

Comment:
Perhaps there could be a way to divide up the end-to-end journey of upload from a typical use case just covering the scope of file-transmission without Upscan? Just so they have a clear idea of where the scope of file-transmission starts and how they could use an alternative file hosting solution. Perhaps the last line of 'MDG asynchronously uses the file' could be clarified/expanded on (if this line is necessary?)